docs/reference/commandline/volume_create.md

```diff
@@ -54,17 +54,25 @@ different volume drivers may do different things (or nothing at all).
 
 The built-in `local` driver on Windows does not support any options.
 
-The built-in `local` driver on Linux accepts options similar to the linux `mount`
-command:
+The built-in `local` driver on Linux accepts options similar to the linux `mount` command. You can provide multiple options using a comma-separated list.
```

Comment:
Maybe we should be really explicit, e.g. "You can provide multiple options by passing the `--opt` flag multiple times. Some mount options (such as the `o` option) take a comma-separated list of options. For example, the following creates a `tmpfs` volume, with a size of 100 megabyte ........"

Perhaps we should add a link to all available options that Linux supports?